lib/components/SearchBar.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+    <div class="searchBarContainer">
+        <img @click="onSubmit($event)" class="searchIcon" src="https://image.flaticon.com/icons/png/128/61/61088.png" />
```

Comment:
We should provide the icons ourselves initially. Can't rely on a cdn. We will let users change the icons later on.

Also, don't set the icon as "src" but set as background rather. We can override the icon easily with css this way.